collectd_plugin/collectd_plugin.py

```diff
@@ -956,7 +962,5 @@ def _should_filter(values, data, ds):
     data = {}
```

Comment:
can confirm: during module import this is empty

registering the config handler, isn't the same as *calling* the config handler - so data is still empty.